### ISSUE TITLE
Switch to typescript

### ISSUE THREAD

**Issue by atybdot** [atybdot/CSI-2024-website#2]
All of this code is in js, while js is great for starting a project but it's not that great for production grade projects.
Convert/Refcator all of the codebase to typescript

**Comment by MindfulLearner** [comment]
Ehy i could  do that need help?

**Comment by atybdot** [comment]
I don't think you should do this, since it's for college club, the club's web-dev team is tasked with these issues,
but thanks.

**Comment by MindfulLearner** [comment]
np! have a nice day!
